tools/debug_tools/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "debug_tools"
+version = "0.0.2"
+authors = ["Lyla <wallamide@gmail.com>"]
+edition = "2021"
+description = "Debugging tools for Emergence"
+publish = false
+license = "MIT OR Apache-2.0"
+
+[lib]
+name = "debug_tools"
+
+[dependencies]
+bevy = "0.9.1"
+bevy_console = { git = "https://github.com/RichoDemus/bevy-console", branch = "main" }
+bevy_ecs_tilemap = { git = "https://github.com/StarArawn/bevy_ecs_tilemap", branch = "main" }
```

Comment:
It's usually a good idea to fix a commit hash to avoid miscompiles in the future

Reply:
this has been removed as a dep since it was used for the tile labels before and I currently don't see a need for it here.